frankapy_control_test_scripts/README.md

```diff
@@ -0,0 +1,51 @@
+## Description
+
+Scripts to test controllers provided in frankapy. Inside the workstation docker, you can find the scripts at /root. We have a implemented a class that subscribes to synched messages of control command and the robot state and uses this to plot system responses to the control inputs. 
```

Comment:
this has not been the case for us at the moment; maybe because of the issue mentioned before.